pkg/trace/telemetry/config.go

```diff
@@ -0,0 +1,63 @@
+package telemetry
```

Comment:
This does not warrant a separate package (just like we don't have separate packages for the other proxies either: profiler, live debugger, etc). It's not a stand alone unit that is used anywhere else. Can we please put everything inside `pkg/trace/api/telemetry.go`?